src/middlewared/middlewared/plugins/kubernetes_linux/update.py

```diff
@@ -31,6 +31,7 @@ class KubernetesModel(sa.Model):
     servicelb = sa.Column(sa.Boolean(), default=True, nullable=False)
     validate_host_path = sa.Column(sa.Boolean(), default=True)
     passthrough_mode = sa.Column(sa.Boolean(), default=False)
+    metrics_server = sa.Column(sa.Boolean(), default=False, nullable=False)
```

Comment:
There's a mismatch between the spellings here and in the migration applied. A column named `metric_server` is being added to database but is being referenced by `metrics_server`.  I think we should have it named as `metrics_server` keeping in line with how k3s itself calls it.